index.css

```diff
@@ -0,0 +1,65 @@
+/* header search bar */
+
+.search__bar__wrapper {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search__bar {
+  margin: 0 auto;
+  width: 100%;
+  height: 3rem;
+  padding: 0 1.25rem;
+  font-size: 1rem;
+  border-radius: 1rem;
+  border: none;
+  background-color: var(--slate-700);
+  outline: none;
+  color: var(--white);
+}
+.search__bar:focus {
```

Comment:
Use beautify extension to have consistent spaces between each class .

Reply:
okay